src/views/virtualmachines/actions/actions.ts

```diff
@@ -108,3 +109,6 @@ export const deleteVM = async (vm: V1VirtualMachine) => {
     resource: vm,
   });
 };
+
+export const expandVirtualMachineSpec = async (vm: V1VirtualMachine) =>
```

Comment:
I think we should use `expandVMSpec` to stay consistent with the other commands here.

Reply:
fixed thanks :)